src/pages/changelog/2025-01-15.mdx

```diff
@@ -0,0 +1,31 @@
+---
+date: 2025-01-25
+title: Signatory UI settings and webhook signing
+---
+import Layout from '../../layouts/mdx';
+
+export default Layout;
+
+## Signatory UI settings
+
+Historically signature UI settings for users have been configured globally on the signature order.
+
+This had some drawbacks as each users may be in different countries and have different ideal languages.
+
+It is now possible to also define UI settings on a per signatory basis, [See example](/signatures/graphql/examples/#signatory-ui-settings)
+
+## Webhook signing
+
+There has up until now not been any way to validate the authenticity of a signature webhook (at the time of the request).
+
+We have relied on the fact that the webhook contained no actual data, but only identifiers, to let clients query our API
+based on the data in the webhook, in that way ensuring only authenticated clients could access signature data.
+
+It was pointed out to us that the webhook could still be used by an attacker to increase the number of requests
+that a well behaving client would have to make our API, potentially triggering rate limits.
+
+We have now added the option to configure a webhook secret resulting in a HMAC-SHA256 signature being
+added to each signature webhook invocation.
```

Comment:
Suggesting minor changes:

Until now, there was no way to validate the authenticity of a signature webhook at the time of the request.

We relied on the fact that webhooks contained no actual data, but only identifiers, allowing clients to query our API based on the data in the webhook. This ensured that only authenticated clients could access signature data.

However, it was brought to our attention that an attacker could use the webhook to increase the number of requests a well-behaving client would have to make to our API, potentially triggering rate limits.

To address this, we introduced the option to configure a webhook secret, which adds an HMAC-SHA256 signature to each signature webhook invocation.